src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@jkotas I am trying to stay 1. Please refers to #8911. It attempts to implement ``RtlVirtualUnwind`` using the existing unwind info as you suggested.